### PR TITLE
Bug/misc

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: lib/clean.py [hunk 1]
```diff
@@ -237,19 +237,15 @@ def clean_races(df: pd.DataFrame, cols: list[str]) -> pd.DataFrame:
...
-            .str.replace(r'\bislande\b', 'islander', regex=True)\
-            .str.replace(r'^a$', 'asian', regex=True)
+            .str.replace(r'\bislande\b', 'islander', regex=True)
         df = standardize_from_lookup_table(df, col, [
             ['black', 'african american', 'black / african american', 'black or african american'],
             ['white'],
             ['hispanic', 'latino'],
             ['native american', 'american indian', 'american indian or alaskan native'],
-            ['asian / pacific islander', 'asian', 'native hawaiian or other pacific islander', 'islander'],
+            ['asian / pacific islander', 'asian', 'native hawaiian or other pacific islander'],
```
Comment (pckhoi) [inline lib/clean.py:246]: Are you sure that "islander" shouldn't be in this group? Doing this might break other scripts.

**Comment by pckhoi** [comment]
Very nice typo corrections!

File: lib/clean.py (hunk 1 above)
Comment (ayyubibrahimi) [reply lib/clean.py]: It should. Thanks.

**Review by pckhoi — APPROVED** [review]
LGTM

**Pull request merged**